src/components/Profile/Profile.js

```diff
@@ -1,24 +1,33 @@
 import styled from 'styled-components';
-import './Profile.css';
 
 function Profile() {
   return (
-    <div className="background__image">
-    <div className="App">
+    <StyledBackground>
+    <ProfileText>
       <header>
         <StyledText>
           Profile Page
         </StyledText>
       </header>
-    </div>
+    </ProfileText>
```

Comment:
I dont think you need this div (ProfileText) here around the header. Just use the header and style it